app/move/controllers/confirmation.js

```diff
@@ -1,7 +1,15 @@
+const { get, map } = require('lodash')
+
 function confirmation(req, res) {
   const { move_type: moveType, to_location: toLocation } = res.locals.move
+  const suppliers = get(res.locals, 'move.from_location.suppliers')
+  const supplierName =
+    suppliers && suppliers.length
+      ? map(suppliers, 'name').join(' and ')
```

Comment:
Is it possible to have more than 2 suppliers? It would result in a chain of `and`. If it's an edge case or otherwise unlikely, ignore this comment.

Reply:
Yep, currently the API supports the possibility of multiple suppliers as it is returned as an array so this was more just to cover that case rather than just always assume it would be the first. 

That could mislead the user if we only tell them the first item in the array.